app.rb

```diff
@@ -0,0 +1,29 @@
+def decode_morse_code()
+  morsecode = ['.-', '-...', '-.-.', '-..', '.', '..-.', '--.', '....', '..', '.---', '-.-', '.-..', '--', '-.', '---',
+               '.--.', '--.-', '.-.', '...', '-', '..-', '...-', '.--', '-..-', '-.--', '--..']
+  morsevalues = %w[a b c d e f g h i j k l m n o p q r s t
+                   u v w x y z]
+  puts "Enter a word to translate to morse code \n e.g: .-   -... --- -..-   ..-. ..- .-.. .-..
+--- ..-.   .-. ..- -... .. . ..."
+  code = gets.chomp
+  code_array = code.split('  ')
+  string = ''
+  code_array.each do |word|
+    word_array = word.split
+    word_array.each do |letter|
+      string += morsevalues[morsecode.index(letter)]
+    end
+    string += ' '
+  end
+  "The translated code is:\n ====> #{string}"
+end
+
+i_have_a_message = true
+
+while i_have_a_message
+  puts decode_morse_code
+  puts 'Would you like to translate another message? (y/n)'
+  i_have_a_message = gets.chomp == 'y'
+end
+
+puts 'Goodbye!'
```

Comment:
- [ ] Hi team, great job implementing a solution to `Decode a Morse code message`, but we do have an issue here, according to the project [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/morse_code.md), we are supposed to implement the solution using methods(reusable methods). For example: 
- `decode_char` that translates Morse code symbols into letters,
- `decode_word` that iterates through all chars from the input word, reuses the `decode_char` method to translate a single char symbol into a letter (DRY rule), combines translated chars into a single word to be returned as the final output
- `decode` splits input sentences by 3 spaces (word delimiter in Morse code), iterates through all words from the input sentence, reuses the `#decode_word` method to translate a single encoded-word into a word (DRY rule), combines translated words into a whole sentence to be returned as the final output. 
Please kindly restructure your code.